### PR TITLE
Update links to new websites - edition March 2020 

### PR DESCRIPTION
**Pull request by archmoj**
This PR updates various links on the repo and in the code mainly in order to point to newer websites
- replace `plot.ly` with `plotly.com` or `chart-studio.plolty.com` depending on the context
- replace `http` refs to `https` 

@plotly/plotly_js 
cc: @VeraZab

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ Then elsewhere in your code:
 var Plotly = require('./path/to/custom-plotly');
 ```
 
-To learn more about the plotly.js module architecture, refer to our [modularizing monolithic JS projects](https://plot.ly/javascript/modularizing-monolithic-javascript-projects/) post.
+To learn more about the plotly.js module architecture, refer to our [modularizing monolithic JS projects](https://plotly.com/javascript/plotly-fundamentals/) post.
```
Comment (nicolaskruchten) [inline README.md]: Let's just remove this line please. The underlying content has been removed.

File: README.md [hunk 2]
```diff
@@ -113,12 +113,12 @@ Have a bug or a feature request? Please first read the [issues guidelines](https
...
 
-Official plotly.js documentation is hosted on [plot.ly/javascript](https://plot.ly/javascript).
+Official plotly.js documentation is hosted on [plotly.com/javascript](https://plotly.com/javascript).
 
-These pages are generated by the Plotly [documentation repo](https://github.com/plotly/documentation/tree/gh-pages) built with [Jekyll](http://jekyllrb.com) and publicly hosted on GitHub Pages.
+These pages are generated by the Plotly [documentation repo](https://github.com/plotly/documentation/tree/gh-pages) built with [Jekyll](https://jekyllrb.com/) and publicly hosted on GitHub Pages.
 For more info about contributing to Plotly documentation, please read through [contributing guidelines](https://github.com/plotly/documentation/blob/source/Contributing.md).
 
-You can also suggest new documentation examples by submitting a [Codepen](http://codepen.io/tag/plotly/) on community.plot.ly with tag [`plotly-js`](http://community.plot.ly/c/plotly-js).
+You can also suggest new documentation examples by submitting a [Codepen](https://codepen.io/tag/plotly/) on community.plot.com with tag [`plotly-js`](https://community.plotly.com/c/api/plotly-js).
```
Comment (nicolaskruchten) [inline README.md]: Let's remove this line plz

File: README.md [hunk 3]
```diff
@@ -113,12 +113,12 @@ Have a bug or a feature request? Please first read the [issues guidelines](https
 
 ## Documentation
 
-Official plotly.js documentation is hosted on [plot.ly/javascript](https://plot.ly/javascript).
+Official plotly.js documentation is hosted on [plotly.com/javascript](https://plotly.com/javascript).
 
-These pages are generated by the Plotly [documentation repo](https://github.com/plotly/documentation/tree/gh-pages) built with [Jekyll](http://jekyllrb.com) and publicly hosted on GitHub Pages.
+These pages are generated by the Plotly [documentation repo](https://github.com/plotly/documentation/tree/gh-pages) built with [Jekyll](https://jekyllrb.com/) and publicly hosted on GitHub Pages.
```
Comment (nicolaskruchten) [inline README.md]: can you update these to replace `documentation` with `graphing-library-docs` please? The new repo is at https://github.com/plotly/graphing-library-docs

File: README.md [hunk 4]
```diff
@@ -128,14 +128,14 @@ Please read through our [contributing guidelines](https://github.com/plotly/plot
 
 * Follow [@plotlygraphs](https://twitter.com/plotlygraphs) on Twitter for the latest Plotly news.
 * Follow [@plotly_js](https://twitter.com/plotly_js) on Twitter for plotly.js release updates.
-* Implementation help may be found on community.plot.ly (tagged [`plotly-js`](http://community.plot.ly/c/plotly-js)) or
+* Implementation help may be found on community.plot.com (tagged [`plotly-js`](https://community.plotly.com/c/plotly-js)) or
   on Stack Overflow (tagged [`plotly`](https://stackoverflow.com/questions/tagged/plotly)).
 * Developers should use the keyword `plotly` on packages which modify or add to the functionality of plotly.js when distributing through [npm](https://www.npmjs.com/browse/keyword/plotly).
-* Direct developer email support can be purchased through a [Plotly Support Plan](https://support.plot.ly/libraries/javascript).
+* Direct developer email support can be purchased through a [Plotly Support Plan](https://plotly.com/chart-studio-help/chart-studio-cloud-support/).
```
Comment (nicolaskruchten) [inline README.md]: Please remove this line.

File: README.md [hunk 5]
```diff
@@ -145,14 +145,14 @@ Open-source clients to the plotly.js APIs are available at these links:
...
+|**MATLAB**| [plotly/matlab-api](https://github.com/plotly/matlab-api) | [plotly/matlab/getting-started](https://plotly.com/matlab/getting-started) |
 |**node.js / Tonicdev / Jupyter notebook**| [plotly/plotly-notebook-js](https://github.com/plotly/plotly-notebook-js) | |
-|**node.js cloud client**| [plotly/plotly-nodejs](https://github.com/plotly/plotly-nodejs) | [plot.ly/nodejs/getting-started](https://plot.ly/nodejs/getting-started) |
-|**Julia**| [plotly/Plotly.jl](https://github.com/plotly/Plotly.jl) | [plot.ly/julia/getting-started](https://plot.ly/julia/getting-started) |
+|**node.js cloud client**| [plotly/plotly-nodejs](https://github.com/plotly/plotly-nodejs) | [plotly/nodejs/getting-started](https://plotly.com/nodejs/getting-started) |
+|**Julia**| [plotly/Plotly.jl](https://github.com/plotly/Plotly.jl) | [plotly/julia/getting-started](https://plotly.com/julia/getting-started) |
 
-plotly.js charts can also be created and saved online for free at [plot.ly/create](https://plot.ly/create).
+plotly.js charts can also be created and saved online for free at [plotly/create](https://chart-studio.plotly.com/create).
```
Comment (nicolaskruchten) [inline README.md]: Please remove this line.

File: README.md [hunk 6]
```diff
@@ -145,14 +145,14 @@ Open-source clients to the plotly.js APIs are available at these links:
...
+|**R / RStudio**| [ropensci/plotly](https://github.com/ropensci/plotly) | [plotly/r/getting-started](https://plotly.com/r/getting-started) |
+|**Python / Pandas / IPython notebook**| [plotly/plotly.py](https://github.com/plotly/plotly.py) | [plotly/python/getting-started](https://plotly.com/python/getting-started) |
+|**MATLAB**| [plotly/matlab-api](https://github.com/plotly/matlab-api) | [plotly/matlab/getting-started](https://plotly.com/matlab/getting-started) |
 |**node.js / Tonicdev / Jupyter notebook**| [plotly/plotly-notebook-js](https://github.com/plotly/plotly-notebook-js) | |
-|**node.js cloud client**| [plotly/plotly-nodejs](https://github.com/plotly/plotly-nodejs) | [plot.ly/nodejs/getting-started](https://plot.ly/nodejs/getting-started) |
-|**Julia**| [plotly/Plotly.jl](https://github.com/plotly/Plotly.jl) | [plot.ly/julia/getting-started](https://plot.ly/julia/getting-started) |
+|**node.js cloud client**| [plotly/plotly-nodejs](https://github.com/plotly/plotly-nodejs) | [plotly/nodejs/getting-started](https://plotly.com/nodejs/getting-started) |
```
Comment (nicolaskruchten) [inline README.md]: Please remove this line.

Comment (archmoj) [reply README.md]: Done in https://github.com/plotly/plotly.js/pull/4694/commits/1fe9c53d66973721d0a512c4570b9c7d612b1f90.

File: README.md (hunk 5 above)
Comment (archmoj) [reply README.md]: Done in https://github.com/plotly/plotly.js/pull/4694/commits/1fe9c53d66973721d0a512c4570b9c7d612b1f90.

File: README.md (hunk 2 above)
Comment (archmoj) [reply README.md]: Done in https://github.com/plotly/plotly.js/pull/4694/commits/1fe9c53d66973721d0a512c4570b9c7d612b1f90.

File: README.md (hunk 1 above)
Comment (archmoj) [reply README.md]: Done in https://github.com/plotly/plotly.js/pull/4694/commits/1fe9c53d66973721d0a512c4570b9c7d612b1f90.

File: README.md (hunk 4 above)
Comment (archmoj) [reply README.md]: Done in https://github.com/plotly/plotly.js/pull/4694/commits/1fe9c53d66973721d0a512c4570b9c7d612b1f90.

File: README.md (hunk 5 above)
Comment (archmoj) [reply README.md]: Done in https://github.com/plotly/plotly.js/pull/4694/commits/df6918cfc8e62bf8fbf581636e49942dd110fe54.

File: README.md (hunk 6 above)
Comment (archmoj) [reply README.md]: Done in https://github.com/plotly/plotly.js/pull/4694/commits/df6918cfc8e62bf8fbf581636e49942dd110fe54.

**Comment by nicolaskruchten** [comment]
:dancer: thanks!

**Pull request merged**